fda-model/compute/ImpactAreaScenarioSimulation.cs

```diff
@@ -496,7 +504,12 @@ private Threshold ComputeDefaultThreshold(ConvergenceCriteria convergenceCriteri
                     {
                         if (_discharge_stage.CurveMetaData.IsNull)
                         {
-                            throw new Exception("A rating curve must accompany a flow-frequency function");
+                            double badThresholdStage = 0;
+                            string message = "A rating curve must accompany a flow-frequency function. An arbitrary threshold is being used.";
```

Comment:
confused why CurveMetaData being null throws this error? I don't see that CurveMetaData holds a rating curve. Is this the wrong message for this case?

Reply:
A stage-discharge function has metadata, so IsNull is a piece of metadata of the stage-discharge function. All of the functions are instantiated with IsNull by default.